solutions/search/search-applications.md

```diff
@@ -54,12 +53,16 @@ Search Applications use [search templates](search-templates.md) to simplify the
 
 ## Get started [search-application-overview-get-started] 
 
-
 ### Option 1: Get started in the UI [search-application-overview-get-started-ui] 
 
 ```{applies_to}
-serverless: unavailable
+serverless: unavailable 
 ```
+:::{warning}
+:applies_to: {"stack": "ga 9.2.0", "ess": "ga 9.2.0"}
```

Comment:
```suggestion
:applies_to: {"stack": "unavailable 9.2.0"}
```

`stack` basically means "everything other than serverless" so that should do the trick here I think

Reply:
the output will say `Unavailable: 9.2` post-release day BTW